README.md

```diff
@@ -47,3 +45,14 @@ python load_registry.py
 
 When a pull request is merged it will then appear at the
 [`fpm registry`](https://fortran-lang.org/packages/fpm).
+
+## How to request help
+
+If you have a great Fortran package you have managed to get onto
+**github** or another **git** repository that you are interested
+in setting up as a registered **fpm** package that adheres to the
+guidelines listed here but terms like **fpm**, **PR**, **toml**, and
+**python** are putting you off from registering your package, please open an
+[issue](https://github.com/fortran-lang/fpm-registry/issues) at the
+**fpm Registry** site and list your Repository and ask for help.  Get as
+far as you can and identify where you got stuck. We will try to help.
```

Comment:
```suggestion
If you have a great Fortran package that you are interested
in registering as an **fpm** package but terms like **fpm**, **PR**, **toml**, and
**python** are putting you off, then please open an
[issue](https://github.com/fortran-lang/fpm-registry/issues) at the
**fpm Registry** site listing your Repository and let us know how we can help. Get as
far as you can and identify where you got stuck. We are always happy to help.
```

This is a *very* long multi-clause sentence; I have removed some redundancy and simplified.